Update playbooks_intro.rst to include dry run option (#159)
ansible/ansible-documentation: docs/docsite/rst/playbook_guide/playbooks_intro.rst

Use the ``--verbose`` flag when running your playbook to see detailed output from successful modules as well as unsuccessful ones.

Running playbooks in check mode 
--------------------------------

> Ansible's check mode, allows you to execute a playbook without applying any alterations to your systems. This feature proves beneficial for testing playbooks before implementing them in a production environment.

Reviewer: ```suggestion
Ansible's check mode allows you to execute a playbook without applying any alterations to your systems. You can use  check mode to test playbooks before implementing them in a production environment.
```

> Ansible's dry run mode, also referred to as check mode, allows you to execute a playbook without applying any alterations to your systems. This feature proves beneficial for testing playbooks before implementing them in a production environment.

Reviewer 2: The actual feature/option is referred to as check mode.

```suggestion
Ansible's check mode, allows you to execute a playbook without applying any alterations to your systems. This feature proves beneficial for testing playbooks before implementing them in a production environment.
```

> To run a playbook in check mode, you can pass the ``-C`` or ``--check`` flag to the ansible-playbook command. For instance:

Reviewer: ```suggestion
To run a playbook in check mode, you can pass the ``-C`` or ``--check`` flag to the ``ansible-playbook`` command::
```

> To perform a playbook in dry run mode, you can employ the -C or --check flag along with the ansible-playbook command. For instance:

Reviewer 2: ```suggestion
To run a playbook in check mode, you can pass the ``-C`` or ``--check`` flag to the ansible-playbook command. For instance:
```

.. code-block:: bash

>     ansible-playbook playbook.yaml --check

Reviewer 2: ```suggestion
    ansible-playbook --check playbook.yaml
```

Executing this command will run the playbook normally, but instead of implementing any modifications, Ansible will simply provide a report on the changes it would have made. This report encompasses details such as file modifications, command execution, and module calls.

> Dry run mode offers a safe and practical approach to examine the functionality of your playbooks without risking unintended changes to your systems. Moreover, it can be utilized as a valuable tool for troubleshooting playbooks that might not be functioning as anticipated.

Reviewer 2: ```suggestion
Check mode offers a safe and practical approach to examine the functionality of your playbooks without risking unintended changes to your systems. Moreover, it is a valuable tool for troubleshooting playbooks that are not functioning as expected.
```
